docs/BR.md

```diff
@@ -1008,6 +1043,62 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+
+The response returned from each corroborating Network Perspective MUST allow the CA to verify:
+- a. the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5; and
+- b. the CA's authority to issue to the requested domain(s), as specified in Section 3.2.2.8.
+
+[Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address) describe the validation methods that require the use of Multi-Perspective Issuance Corroboration and how a Network Perspective can corroborate the outcomes determined by the Primary Network Perspective. A CA MAY use corroborating evidence for CAA record compliance for a period equal to the period of validity for Validation of Domain Authorization or Control specified in [Section 4.2.1](#421-performing-identification-and-authentication-functions).
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives). The network infrastructure providing Internet connectivity to a Network Perspective MAY be administered by the same organization providing the computational services required to operate the Network Perspective.
+
+CAs MAY immediately retry Multi-Perspective Issuance Corroboration using the same validation method or an alternative method (e.g., a CA can immediately retry validation using "Email to DNS TXT Contact" if "Agreed-Upon Change to Website - ACME" does not corroborate the outcome of Multi-Perspective Issuance Corroboration). When retrying Multi-Perspective Issuance Corroboration, CAs MUST NOT rely on corroborations from previous attempts. There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+The "Quorum Requirements" Table describes quorum requirements related to Multi-Perspective Issuance Corroboration. Network Perspectives are considered distinct when the straight-line distance between the two States, Provinces, or Countries they reside in is at least 500 km. Network Perspectives are considered "remote" when they are distinct from the Primary Network Perspective and the other Network Perspectives represented in a quorum.
+
+Table: Quorum Requirements
+
+| # of Distinct Remote Network Perspectives Used | # of Allowed non-Corroborations |
+| --- | --- |
+| 2-5 |  1  |
+| 6+  |  2  |
+
+Network Perspectives performing Multi-Perspective Issuance Corroboration:
+
+- MUST:
+    - Forward all Internet traffic via a network or set of networks that filter all RPKI-invalid BGP routes as defined by RFC 6811. 
```

Comment:
The scope of this MUST requirement may require additional clarification to prevent ambiguity. For example, are "remote" network perspectives required to be RPKI filtered, or also the current "primary" network perspectives (which could be existing infrastructure) which might not be the intention and could have a big impact? In the latter case, as of when does it apply, as of the effective date of the ballot or as of the MPIC requirement date?

Reply:
Great points. The intent was never to require the primary perspective to adopt RPKI filtering. This could require the selection of a new ISP, and some CAs may be in a region that is not well serviced by an RPKI-filtering ISP.

The intent is to require the deployment of new infrastructure (which is inherently remote giving CAs more liberty in selection of providers and locations) to be done using RPKI-filtering ISPs.

As for the effective date, I think this really only makes sense as a security improvement after June 15, 2025 at which point MPIC is actually capable of blocking issuance. I could see the argument of making the date earlier to push CAs to deploy infrastructure in RPKI-filtering ISPs from day one, but I personally feel making it as late as possible makes sense to give maximum time for implementation (also RPKI deployment continues to grow so there may be providers that are planning to deploy RPKI-filtering but are have not fully by the earlier date).

Here is a change that might clarify some of these points.

```
Network Perspectives performing Multi-Perspective Issuance Corroboration other than the Primary Network Perspective:

- By June 15, 2025 MUST :
    - Forward all Internet traffic via a network or set of networks that filter all RPKI-invalid BGP routes as defined by RFC 6811. 
```

Reply:
Sorry for the delayed response, @chrisbn and @birgelee.

In my opinion, nothing in the draft language prevents a CA owner from promoting a remote network perspective to act as the Primary Perspective. This proposal does not change expectations related to how domain validation takes place. If a CA owner were to promote a remote perspective to the role of a primary perspective, all requirements that exist for domain validation today apply. 

It also seems that delaying the adoption of filtering RPKI-invalid routes adds complexity and creates opportunity for misalignment with a well-defined future state with limited benefit. For example, this approach might necessitate re-work given it's beyond the CA owner's control as to whether an existing remote perspective's ISP adopts RPKI (or adopts it on a specific timeline).  Finally, and in my opinion, the value of the "monitoring mode" period (until June 15, 2025) is to most closely simulate the future state where enforcement (i.e., blocking) is required. This period is intended to allow CA owners to identify potential challenges and fine-tune implementations before they can block issuance. By failing to adopt the full set of future requirements, this period will not help the ecosystem realize its intended value proposition.

Reply:
@ryancdickson I think that is a good point about ensuring monitoring mode performs as close to the enforcing deployment as possible. Would it make sense to move this date to be the start of monitoring mode?

I personally do like the date requirement being monitoring mode and not just the adoption time of this ballot. Google Trust Services and Let's Encrypt have systems that qualify as remote network perspectives and, even though there is no other normative requirement before the monitoring mode date, it seems to me that having no specified date on this line could potentially require them to immediately change their systems before adoption of this ballot. Having the monitoring mode date on this line makes it clear they have the same timeline as newly deployed systems to ensure compliance with this requirement.

I also feel a date earlier than monitoring mode for the RPKI-filtering requirement is unenforceable as a CA may decide to **not** issue a certificate for any reason of its choosing. This could be due to a multi-perspective check from non-RPKI filtering perspectives. Since there is no other normative requirement in this ballot before monitoring mode, nothing else in this ballot would be violated, so I feel it is somewhat strange to say a CA is in violation because it went above and beyond and performed this extra check.

@ryancdickson would something like this make sense:

```
Network Perspectives performing Multi-Perspective Issuance Corroboration other than the Primary Network Perspective:

- By December 15, 2024 MUST :
    - Forward all Internet traffic via a network or set of networks that filter all RPKI-invalid BGP routes as defined by RFC 6811. 
```

Reply:
@birgelee - That's a fair perspective regarding the potential for unintentional impact on CA owners who might have already adopted the practice before the ballot becomes effective. 

The way I interpret the requirements (and how I tried to write them):
- The requirements described in the proposal could be added to the BRs at any time.
- Between June 15, 2024, and December 14, 2024: CAs are encouraged but ultimately may optionally choose to support MPIC as described in the proposal. Because MPIC is optional, CAs would not be penalized for an "incomplete" implementation that does not align with subsequent enforcement milestones (like the one described in the next bullet). In this case, if an existing implementer isn't filtering RPKI-invalid routes, that shouldn't be considered a violation of the BRs (because the practice of performing MPIC as described in the ballot is recommended, not required). 
- Beginning December 15, 2024: MPIC is required (to include those requirements related to the quorum and network perspective security controls - like filtering RPKI-invalid routes); however, CAs are not yet required to block issuance if the primary perspective is not corroborated via MPIC checks. 

To that end, the filtering of RPKI-invalid routes would become effective on December 15, 2024 - but the same is true for all other related requirements. This, for example, is why we noted 12/15/24 as the "Relevant Date" for MPIC included in 1.2.2.

Given the above, does that change your opinion?

Reply:
If this is how the text is interpreted without any date specified on this line, I am fine removing any date reference here.

@ryancdickson Do you have any thoughts on the "other than the Primary Network Perspective" language change? My understanding was that the ballot draft was not to change the existing DV infrastructure but simply require new infrastructure to have RPKI filtering.

Reply:
@birgelee - It feels unnecessary to me.

I wouldn't consider the Primary Perspective as "performing" MPIC (instead, the remote perspectives are performing MPIC to corroborate the determinations made by the Primary Perspective) - meaning there's no need to further exclude the Primary Perspective from the scope of these requirements (as it already is excluded).

Reply:
Would simply changing
> Network Perspectives performing Multi-Perspective Issuance Corroboration:
to
> Remote Network Perspectives performing Multi-Perspective Issuance Corroboration:
Be enough to ensure it wont impact the Primary Perspective?

Reply:
Added in https://github.com/ryancdickson/staging/commit/cb88b7d7bec8717e9903f6c91adbf0171f68681b